templates/src/components/App/App.js

```diff
@@ -20,6 +19,8 @@ import layout from '../../util/layout';
 import usePassiveEvent from '../../util/use-passive-event';
 import checkProps from '../../util/check-props';
 
+const RotateScreen = detect.isMobile ? lazy(() => import('../../components/Rotate/Rotate')) : null;
```

Comment:
I added the ternary `if` so `<RotateScreen>` could just be added to the `render` method (w/o checking there if `detec.isMobile`). But u are right that we need to wrap it in `<Suspense>` and for that u need to check again if `detect.isMobile`, so I think we can safely just change this to:
```suggestion
const RotateScreen = detect.isMobile && lazy(() => import('../../components/Rotate/Rotate'));
```
What do u think?